simulator/source/Combat_simulator.cpp

```diff
@@ -1522,6 +1590,10 @@ void Combat_simulator::normal_phase(Sim_state& state, bool mh_swing)
         {
             use_ww = std::max(time_keeper_.mortal_strike_cd(), 100) > config.combat.whirlwind_bt_cooldown_thresh;
         }
+        if (use_devastate_)
+        {
+            use_ww = true;
+        }
```

Comment:
This is ineffective: either Devastate has happened, in which case we never get here; or it yielded to Whirlwind, in which case there's nothing to do, either.

Reply:
Yeah, when I created that to mimic the checks for MS and BT, I realized it couldn't have any effect since devastate has no cooldown. I meant to take a pass back at that and make sure there was in fact nothing to check there but didn't get around to it. I went ahead and removed it.